test/failures_testsuite/utils/s3.py

```diff
@@ -79,6 +80,16 @@ def vm_robot(self):
             self._vm_robot = j.clients.zrobot.robots['demo_vm_robot']
         return self._vm_robot
 
+    @property
+    def vm_host_robot(self):
+        """
+        zrobot client of the node  that hosts the zos VM which hosts the minio container
+        """
+        if self._vm_host_robot is None:
+            j.clients.zrobot.get('demo_vm_host_robot', data={'url': "http://%s:6600" % self.vm_host.public_addr}) # 'god_token_': ''
+            self._vm_host_robot = j.clients.zrobot.robots['demo_vm_host_robot']
+        return self._vm_host_robot
```

Comment:
u can't do anything with this robot without god_token, u can use get_god_token method from the faliures file